tests/dashboards/api_tests.py

```diff
@@ -955,12 +956,14 @@ def test_export(self):
         self.login(username="admin")
         argument = [1, 2]
         uri = f"api/v1/dashboard/export/?q={prison.dumps(argument)}"
-        rv = self.get_assert_metric(uri, "export")
+
+        # freeze time to ensure filename is deterministic
+        with freeze_time("2020-01-01T00:00:00Z"):
+            rv = self.get_assert_metric(uri, "export")
+            headers = generate_download_headers("json")["Content-Disposition"]
+
         self.assertEqual(rv.status_code, 200)
-        self.assertEqual(
-            rv.headers["Content-Disposition"],
-            generate_download_headers("json")["Content-Disposition"],
-        )
+        self.assertEqual(rv.headers["Content-Disposition"], headers)
```

Comment:
This test failed with me once, because the call to `get_assert_metric` and the call to `generate_download_headers` can result in different timestamps. Using `freeze_time` here will make the test always succeed.

Reply:
nit: We are currently preferring `assert rv.headers["Content-Disposition"] == headers`. More inline with pytest

Reply:
Will do!